lib/cbus_elixir/app/app.ex

```diff
@@ -70,10 +72,12 @@ defmodule CbusElixir.App do
   def update_speaker(%Speaker{} = speaker, attrs) do
     speaker
     |> Speaker.changeset(attrs)
+
```

Comment:
Why the blank lines? Try running `mix format`

Reply:
No clue. I removed the meeting stuff and surrounding git comments about conflicts.
`mix format` did not work globally. had to provide the specific file. Updated and pushed.